.editorconfig

```diff
@@ -0,0 +1,331 @@
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+indent_size = 4
+indent_style = space
+insert_final_newline = true
+max_line_length = 120
```

Comment:
Will it affect JSON files?

Reply:
No

Reply:
We have specific json properties, which gives json lines possibility to be as long as needed